SI/Sem.03/Pract.03/Task 04 - 06/Task06.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+    int n, ogSum = 0, repSum = 0;
```

Comment:
ogSum/repSum , I would suggest to review the names and pick a more suitable ones.